microhapdb/tests/test_marker.py

```diff
@@ -687,9 +695,7 @@ def test_standardize_ids_incomplete_prefixes():
 
 def test_locus_length():
     loci = defaultdict(Locus)
-    for marker in Marker.objectify(microhapdb.markers):
+    for marker in Marker.objectify(microhapdb.markers, delta=0, minlen=0):
         loci[marker.locus].markers.append(marker)
     for locus in loci.values():
-        # This is the length of the Fasta representation of the sequence, not the sequence itself,
-        # but...close enough. 🙃
-        assert len(locus.fasta) < 700, locus.name
+        assert len(locus.target_seq) <= 610, locus.name
```

Comment:
Forgot about the `locus.target_seq` method, which is much better suited for this test.